build/tools/BuildImage.bat

```diff
@@ -87,16 +87,16 @@ if "%SolutionDir%"=="" (
 REM
 echo Export packages to one directory
 REM
-set BSPPKG_DIR=%REPO_BUILD_ROOT%\solution\iMXPlatform\Build\FFU\bspcabs
+set BSPPKG_DIR=%REPO_BUILD_ROOT%\solution\iMXPlatform\Build\FFU\bspcabs\%PLATFORM%\%Configuration%
 set PKG_VER=1.0.0.0
 if not exist %BSPPKG_DIR% ( mkdir %BSPPKG_DIR% )
 
-if not exist %REPO_BUILD_ROOT%\..\..\b\%Configuration%\ARM (dir /S /B %REPO_BUILD_ROOT%\solution\iMXPlatform\Build\%Configuration%\ARM\*.cab > filelist.txt)
-if exist %REPO_BUILD_ROOT%\..\..\b\%Configuration%\ARM (dir /S /B %REPO_BUILD_ROOT%\..\..\b\%Configuration%\ARM\*.cab > filelist.txt)
+if not exist %REPO_BUILD_ROOT%\..\..\b\%PLATFORM%\%Configuration% (dir /S /B %REPO_BUILD_ROOT%\solution\iMXPlatform\Build\%PLATFORM%\%Configuration%\*.cab > filelist.txt)
```

Comment:
```suggestion
if not exist %REPO_BUILD_ROOT%\..\..\b\%Configuration%\%PLATFORM% (dir /S /B %REPO_BUILD_ROOT%\solution\iMXPlatform\Build\%PLATFORM%\%Configuration%\*.cab > filelist.txt)
```

Reply:
Recommend adding a comment explaining which config is specific to our CI build server for future reference.